treeherder/intermittents_commenter/commenter.py

```diff
@@ -117,38 +119,41 @@ def generate_bug_changes(self, startday, endday, alt_startday, alt_endday):
                             }
                            }
 
-            if bug_info is not None and whiteboard != bug_info['whiteboard']:
-                bug_changes['changes']['whiteboard'] = whiteboard
+            if change_whiteboard:
+                bug_changes['changes']['whiteboard'] = change_whiteboard
+
             if change_priority:
                 bug_changes['changes']['priority'] = change_priority
 
             all_bug_changes.append(bug_changes)
 
         return all_bug_changes
 
-    def check_needswork_owner(self, change_priority, bug_info, whiteboard):
+    def check_needswork_owner(self, change_priority, bug_info):
+        change_priority = None
```

Comment:
This means that the passed in ``change_priority`` doesn't get used.  This creates a new instance which gets passed back.  So you don't need to pass that value in.  Is that intended?

Reply:
Ah, no - thanks for catching that! No, it doesn't need to be passed in actually. It's only going to be reassigned or not at all. Although, since I already have it initialized as None in the outer scope, does it make sense to pass it in or does that just make it more confusing?

Reply:
Ok, gotcha.  I think then just skip passing it in.  And go ahead and initialize it the way you're doing it.  So the only change here would be to remove the param from the function def (and then don't pass it in when you call it, of course).  Sounds like it's working correctly now, then.